languages/fsharp/exercises/concept/numbers-floating-point/.docs/instructions.md

```diff
@@ -0,0 +1,45 @@
+In this exercise you'll be working with savings accounts. Each year, the balance of your savings account is updated based on its interest rate. The interest rate your bank gives you depends on the amount of money in your account (its balance):
+
+- -3.213% for a negative balance.
+- 0.5% for a positive balance less than `1000` dollars.
+- 1.621% for a positive balance greater or equal than `1000` dollars and less than `5000` dollars.
+- 2.475% for a positive balance greater or equal than `5000` dollars.
```

Comment:
I think we could remove the word "positive"  on these two.

Reply:
We could, but I sort of like the symmetry with the previous line, as it makes it visually clear that there belong to the same category: positive balances.

Reply:
Cool. I went the other way of making each line stand out a bit more, but your logic makes just as much sense :)

Reply:
:)